### ISSUE TITLE
not working properly in VSCode1.12.1 

### ISSUE THREAD

**Issue by nicolazj** [ionutvmi/spacegray-vscode#5]
After update vscode to latest version, this theme seems broken.

platform: mac
vscode version: 1.12.1

**Comment by DougTy** [comment]
I'm also experiencing this. Unfortunately I have no idea how to actually fix the theme itself, but I've managed to get *most* of the missing colours fixed in my VSCode preferences as a temporary solution:

```
"workbench.colorCustomizations":
{        
    "tab.activeBackground": "#2b303b",
    "editor.background": "#2b303b",
    "editor.lineHighlightBorder": "#2b303b",
    "editor.lineHighlightBackground": "#363d48",
    "editor.selectionBackground": "#4f5b66"
},
```

**Comment by coltonneil** [comment]
Here are all of the settings changes I made for ocean dark:

"workbench.colorCustomizations": {
        "editor.background": "#2B303B",
        "sideBar.background": "#232830",
        "sideBarTitle.foreground": "#C2D6DC",
        "activityBar.foreground": "#C2D6DC",
        "activityBar.background": "#1C1F26",
        "statusBar.background": "#1C1F26",
        "sideBarSectionHeader.background": "#232830",
        "scrollbarSlider.activeBackground": "#343d46",
        "tab.inactiveBackground": "#1C1F26",
        "tab.activeBackground": "#2B303B",
        "tab.border": "#1C1F26",
        "tab.activeForeground": "#C2D6DC",
        "tab.inactiveForeground": "#C2D6DC",
        "editorGroupHeader.tabsBackground": "#1C1F26",
        "list.hoverBackground": "#333846",
        "input.background": "#3B4051",
        "scrollbarSlider.background": "#343d46",
        "scrollbarSlider.hoverBackground": "#343d46",
        "editor.findMatchHighlightBackground": "#264F78"
    },

**Comment by ionutvmi** [comment]
This has been fixed in 1.2.1 for all themes.

Please let me know if you still have issues.
